src/domTree.js

```diff
@@ -29,23 +29,31 @@ const createClass = function(classes: string[]): string {
 };
 
 // To ensure that all nodes have compatible signatures for these methods.
-interface VirtualDomNode {
+interface VirtualNodeInterface {
     toNode(): Node;
     toMarkup(): string;
 }
 
-export interface CombinableDomNode extends VirtualDomNode {
-    tryCombine(sibling: CombinableDomNode): boolean;
+interface HtmlDomInterface extends VirtualNodeInterface {
+    classes: string[];
+    height: number;
+    depth: number;
+    maxFontSize: number;
+
+    tryCombine(sibling: HtmlDomNode): boolean;
```

Comment:
I don't know about flowtype naming convention, but should the name of an interface contain the word `Interface`?

Reply:
I didn't see any guidelines regarding that.

I originally didn't using `*Interface`. But this file defines quite a few new types, a number of which are closely related to these interfaces; so it helped to make the interfaces stand out. Note that none of the interfaces are actually exported from this file, so changing the convention here in the future should be very easy as well.